### DISCUSSION TITLE
Verbrauchsdaten auslesen und Visualisieren

### DISCUSSION THREAD

**Discussion by ZeroDown08152** [camueller/SmartApplianceEnabler#429] [Allgemein]
Hallo zusammen,
ich stelle mit ein kleines Display im Wohnbereich vor, wo ich (ähnlich einer Wetterstation) die Daten auf einen Blick habe vom SMA HM.
Kann man das mit dem SMA SAE auslesen, dass ich es dann per Arduino abgreife und auf einem einfachen Display ausgebe?
Wie komme ich auf die Daten?
Erzeugung, Verbrauch, Batterieladung, Netzbezug

Danke für eure Hilfe
Gruß
Marco

**Comment by camueller** [comment]
> Kann man das mit dem SMA SAE auslesen

Nein.
